### ISSUE TITLE
feat: Creeat PR template meeting Communityy standards

### ISSUE THREAD

**Issue by FrancescoXX** [FrancescoXX/contentbox#44]
### What feature?

Creeat PR template meeting Communityy standards

### Record

- [X] I have checked the existing [issues](https://github.com/FrancescoXX/contentbox/issues)
- [ ] I want to work on this issue

**Comment by Dun-sin** [comment]
Hi Fran, can I do this?

**Comment by kumarsonsoff3** [comment]
Sorry Francesco, I got a bit late on this. But no worries, I would provide the `.md` content to that, and Dunsin can fix this after getting assigned! :)

Anything you want to add or subtract, @FrancescoXX 


```
<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->

## Fixes Issue

<!-- Remove this section if not applicable -->

<!-- Example: Closes #31 -->

## Changes proposed

<!-- List all the proposed changes in your PR -->

<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
<!--
[x] - Correct; marked as done
[X] - Correct; marked as done
[ ] - Not correct; marked as **not** done
-->

## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots

<!-- Add all the screenshots which support your changes -->

## Note to reviewers

<!-- Add notes to reviewers if applicable -->

```

**Comment by FrancescoXX** [comment]
assigned to @kumarsonsoff3
